pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -0,0 +1,174 @@
+package consensus
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/jonboulle/clockwork"
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+)
+
+var info = capabilities.MustNewCapabilityInfo(
+	"ocr3",
+	capabilities.CapabilityTypeConsensus,
+	"OCR3 consensus exposed as a capability.",
+	"v1.0.0",
+)
+
+type capability struct {
+	services.StateMachine
+	capabilities.CapabilityInfo
+	store  *store
+	cancel func()
+
+	clock clockwork.Clock
+
+	newExpiryWorkerCh chan *request
+}
+
+func newCapability(s *store, clock clockwork.Clock) *capability {
+	o := &capability{
+		CapabilityInfo:    info,
+		store:             s,
+		newExpiryWorkerCh: make(chan *request),
+		clock:             clock,
+	}
+	return o
+}
+
+func (o *capability) Start(ctx context.Context) error {
+	return o.StartOnce("OCR3Capability", func() error {
+		innerCtx, c := context.WithCancel(ctx)
```

Comment:
The `Start` ctx is not meant to be held long term. Typically we would store a `StopChan` on `capabilty` and then derive contexts from that as needed.